locale/en/LC_MESSAGES/django.po

```diff
@@ -1200,83 +1393,200 @@ msgstr ""
 msgid "Thanks for submitting your story!"
 msgstr ""
 
+#: gem/templates/rwanda/base.html:208 gem/templates/springster/base.html:314
+#: gem/templates/springster/patterns/basics/copyright/copyright-terms.html:7
+#: temp/temp_ve/core/templates/patterns/basics/copyright/copyright-terms.html:7
+msgid "Terms &amp; Conditions"
+msgstr ""
+
+#: gem/templates/rwanda/core/banner_page.html:37
+#: gem/templates/rwanda/core/banner_page.html:41
+#: gem/templates/rwanda/core/tags/bannerpages.html:18
+#: gem/templates/rwanda/core/tags/bannerpages.html:47
+#: gem/templates/springster/core/banner_page.html:37
+#: gem/templates/springster/core/banner_page.html:41
+#: gem/templates/springster/core/tags/bannerpages.html:17
+#: gem/templates/springster/core/tags/bannerpages.html:46
+#: gem/templates/springster/patterns/components/banners/main-banner.html:25
+msgid "Click Here"
+msgstr ""
+
+#: gem/templates/rwanda/core/main.html:7
+msgid ""
```

Comment:
@sewagodimo can you please fix these too and remove the CSS from the trans block